<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationSetupWidget</class>
 <widget class="QWidget" name="AnimationSetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Animation Setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5" stretch="70,30">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="60,40">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4" stretch="75,20,5">
         <item>
          <widget class="PivotView" name="gvPivot"/>
         </item>
         <item>
          <widget class="QListWidget" name="lbImages">
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="movement">
            <enum>QListView::Snap</enum>
           </property>
           <property name="resizeMode">
            <enum>QListView::Fixed</enum>
           </property>
           <property name="layoutMode">
            <enum>QListView::SinglePass</enum>
           </property>
           <property name="viewMode">
            <enum>QListView::IconMode</enum>
           </property>
           <property name="uniformItemSizes">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
           <item>
            <widget class="QPushButton" name="bChange">
             <property name="text">
              <string>Change Spritesheet</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bAdd">
             <property name="text">
              <string>Add Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bRemove">
             <property name="text">
              <string>Remove Image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bAnimate">
             <property name="text">
              <string>Animate</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="95,5">
         <item>
          <widget class="QTableWidget" name="tSheetProps">
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Property</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="bSetPivot">
             <property name="text">
              <string>Set Pivot</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bResetPivot">
             <property name="text">
              <string>Reset Pivot</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bCancelPivot">
             <property name="text">
              <string>Cancel Pivot</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabs">
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PivotView</class>
   <extends>QGraphicsView</extends>
   <header>PivotView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
